flow/TLSConfig.actor.cpp

```diff
@@ -741,9 +741,9 @@ std::string getX509Name(const X509_NAME* name) {
 	}
 	X509_NAME_print_ex(out.get(), name, /* indent= */ 0, /* flags */ XN_FLAG_ONELINE);
 	unsigned char* rawName = nullptr;
```

Comment:
Why `unsigned char` vs just `char` (that BIO_get_mem_data expects)? It is a matter of interpretation and I don't think it affects functionality of this code, but curious why we have unsigned here.

Reply:
Well, tbh, it is just my habit, I prefer unsigned char over char. This may *not* be a good habit since I recalled that STL people said they made a bad choice setting `std::vector` index to be unsigned.

Reply:
I generally use plain `char` if I'm storing some form of readable text e.g. name. If I'm using char as means to store raw bytes (to represent integer/number) and then I want to do math on it (arithmetic, bit shifting, etc.), that's where signed vs unsigned can cause some difference in behavior.

Since you used `unsigned char`, I got curious if there's some math involved and sounds like no. And since it's a simple text use-case, I don't think compiler will generate different code for `char` vs `unsigned char`, so we should be good.